docs/understanding_reports.md

```diff
@@ -0,0 +1,40 @@
+# Understanding statistics
+
+This document is here to help you understand reports from [the nightly publishing job](https://github.com/open-vsx/publish-extensions/actions/workflows/publish-extensions.yml).
+
+## How to gather these statistics
+
+If you click on a job in the GitHub Actions tab, there is an `Artifacts` section at the bottom of the page, from which you can download the `report`, which after unarchiving reveals two files: `result.log` and `stat.json`.
```

Comment:
do you also want to mention the fact that `report.zip` also contains `meta.json`?